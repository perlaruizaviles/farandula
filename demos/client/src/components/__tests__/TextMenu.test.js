import React from 'react';
import {shallow, mount} from 'enzyme';
import TextMenu from '../TextMenu';
import {Map, List} from 'immutable';

it('renders without crashing (shallow)', () => {
  shallow(<TextMenu options={List(['x'])} selected={'x'} />);
});

it('renders without crashing (full)', () => {
  mount(<TextMenu options={List(['x'])} selected={'x'} />);
});