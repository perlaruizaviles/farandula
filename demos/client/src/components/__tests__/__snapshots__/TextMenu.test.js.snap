exports[`Rendering TextMenu  Should create an snapshot for TextMenu 1`] = `
initialize {
  "0": Object {
    "attribs": Object {},
    "children": Array [
      Object {
        "attribs": Object {
          "class": "ui text menu",
        },
        "children": Array [
          Object {
            "attribs": Object {
              "class": "item",
            },
            "children": Array [
              Object {
                "data": "Round Trip",
                "next": null,
                "parent": [Circular],
                "prev": null,
                "type": "text",
              },
            ],
            "name": "a",
            "next": Object {
              "attribs": Object {
                "class": "item",
              },
              "children": Array [
                Object {
                  "data": "One Way",
                  "next": null,
                  "parent": [Circular],
                  "prev": null,
                  "type": "text",
                },
              ],
              "name": "a",
              "next": null,
              "parent": [Circular],
              "prev": [Circular],
              "type": "tag",
            },
            "parent": [Circular],
            "prev": null,
            "type": "tag",
          },
          Object {
            "attribs": Object {
              "class": "item",
            },
            "children": Array [
              Object {
                "data": "One Way",
                "next": null,
                "parent": [Circular],
                "prev": null,
                "type": "text",
              },
            ],
            "name": "a",
            "next": null,
            "parent": [Circular],
            "prev": Object {
              "attribs": Object {
                "class": "item",
              },
              "children": Array [
                Object {
                  "data": "Round Trip",
                  "next": null,
                  "parent": [Circular],
                  "prev": null,
                  "type": "text",
                },
              ],
              "name": "a",
              "next": [Circular],
              "parent": [Circular],
              "prev": null,
              "type": "tag",
            },
            "type": "tag",
          },
        ],
        "name": "div",
        "next": null,
        "parent": null,
        "prev": null,
        "root": [Circular],
        "type": "tag",
      },
    ],
    "name": "root",
    "next": null,
    "parent": null,
    "prev": null,
    "type": "root",
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xmlMode": false,
  },
}
`;
